README.md

```diff
@@ -1,46 +1,81 @@
+
+![Example how to turn on the network pane](edge-devtools-for-vscode-settings-network.png)
+
+You can see an example of the change in the following screencast. After restart, you get an extra tab with network functionality.
+
+![Example how to turn on the network pane](basic_usage(network_and_headless).gif)
+
+#### Turning on Headless Mode
+
+By default, the extension will launch the browser in its own window. This means you get an extra browser icon in your task bar and you need to turn on casting to see the browser inside the editor. You can also choose "headless mode" to not have the browser open in an own window, but embed itself directly into Visual Studio Code.
+
+**Note**: In the past we had issues on Macintosh computers where the Microsoft Edge instance reported itself as "inactive" when the window wasn't visible. Using headless mode fixes that problem.
+
+![Example how to turn on the network pane](edge-devtools-for-vscode-settings-headless.png)
+
+You can see an example of the change in the following screencast:
+
+![Example how to turn on the headless mode](basic_usage(headless).gif)
+
+### Using the tools
+
+The extension operates in two modes - it can launch an instance of Microsoft Edge navigated to your app, or it can attach to a running instance of Microsoft Edge. Both modes requires you to be serving your web application from local web server, which is started from either a Visual Studio Code task or from your command-line. Using the `url` parameter you simply tell Visual Studio Code which URL to either open or launch in the browser.
+
+You can now use the high-fidelity tools to tweak your CSS and inspect network calls and go directly back to your code without leaving the editor.
+
+![Microsoft Edge Tools - Demo](basic_usage.gif)
+
+#### Opening source files from the Elements tool
 
-### Opening source files from the Elements tool
 One of the features of the Elements tool is that it can show you what file applied the styles and event handlers for a given node.
 
 ![Microsoft Edge Tools - Links](links.png)
 
-The source files for these applied styles and attached event handlers appear in the form of links to a url specified by the browser. Clicking on one will attempt to open that file inside the VS Code editor window. Correctly mapping these runtime locations to actual files on disk that are part of your current workspace, may require you to enable source maps as part of your build environment.
+The source files for these applied styles and attached event handlers appear in the form of links to a url specified by the browser. Clicking on one will attempt to open that file inside the Visual Studio Code editor window. Correctly mapping these runtime locations to actual files on disk that are part of your current workspace, may require you to enable source maps as part of your build environment.
 
 An example webpack configuration for sass and typescript is given below:
+
 ```javascript
```

Comment:
I have not tried this but what about something more basic like:
```
"vscode-edge-devtools.pathMapping": {
            "/": "${workspaceFolder}",
            "/css":"${workspaceFolder}/customCss"
        }
```

Reply:
Let's try it and add it once we verified it.